### PR TITLE
Add Prestigio PSP5504DUO

### PR DESCRIPTION
**Pull request by Miro221022**
Pleasse ad this device

### CODE REVIEW

File: data/devices/alcatel.yml [hunk 1]
```diff
@@ -34,7 +34,88 @@
...
+      - fbdev
+    brightness_path: /sys/devices/platform/leds-mt65xx/leds/lcd-backlight/brightness
+    max_brightness: 255
+    cpu_temp_path:  /sys/devices/virtual/thermal/thermal_zone1/temp
+    theme: portrait_hdpi
+
+- name: Alcatel Pixi 4(7)
+  id: Pixi4-7
+  codenames:
+    - Pixi4-7
```
Comment (Miro221022) [inline data/devices/alcatel.yml]: Pixi4-7_3G

**Comment by yshalsager** [comment]
@Miro221022 you must add your edited yml file to separate branch, like this your device won't be added

**Comment by chenxiaolong** [comment]
Please leave out the travis and README changes. Thanks!

**Comment by chenxiaolong** [comment]
Closing old PRs. If you would like this merged, please submit the changes for the devices in separate PRs (as @yshalsager mentioned) and remove the travis changes. Thanks!

**Pull request closed without merging**